include/openssl/ec.h

```diff
@@ -84,6 +84,9 @@ typedef enum {
     POINT_CONVERSION_HYBRID = 6
 } point_conversion_form_t;
 
+const char *OSSL_EC_curve_nid2name(int nid);
+int OSSL_EC_curve_name2nid(const char *name);
```

Comment:
I can understand the purpose of `OSSL_EC_curve_nid2name()`, for apps that use NIDs and want the corresponding name to use with `OSSL_EC_keygen()`...  but the other one, no.